i18n/fr_FR.csv

```diff
@@ -0,0 +1,448 @@
+"If set to 'No' the order is not invoiced.","Si 'Non' est sélectionné, aucune facture ne sera créée dans Magento."
+"Payment success status without invoice","État de réussite du paiement sans facture"
+"Redirect url after 'Success'","URL de redirection après 'Succès'"
+"Redirect after successful payments.","Rediriger après des paiements réussis."
+"Redirect url after 'Failure'","URL de redirection après 'Échec'"
+"Redirect after failed payments.","Rediriger après des paiements échoués."
+"'Canceled' state on failed payment","'Canceled' status bij mislukte betalingen"
+"Orders will stay open after failed payments.","État 'Annulé' en cas d'échec de paiement"
+"Debug email","E-mail de débogage"
+"Transaction data send as raw data.","Les données de transaction sont envoyées sous forme de données brutes."
+"Fee percentage mode","Mode de pourcentage de frais"
+"This setting only applies to percentage-based Buckaroo fees.<br />Please choose over which amount the fee percentage is calculated.","Ce paramètre s'applique uniquement aux frais Buckaroo basés sur un pourcentage.<br />Veuillez choisir sur quel montant le pourcentage de frais est calculé."
+"Please enter these fields as they appear on your bank account.","Veuillez saisir ces champs tels qu'ils apparaissent sur votre compte bancaire."
+"You will receive an email with further payment instructions when you place your order.","Vous recevrez un e-mail avec d'autres instructions de paiement lorsque vous passerez votre commande."
+"Legend","Légende"
+"Payment method","Mode de paiement"
+"Disabled","Désactivé"
+"Test Mode","Mode d'essai (TEST)"
+"Live Mode","Mode en direct (LIVE)"
+"Magento &amp; 3rd party version compatibility","Magento &amp; Compatibilité des versions tierces"
+"Magento version compatibility","Compatibilité des versions de Magento
+"Magento Community Edition version","Version de l'édition communautaire de Magento"
+"Magento Enterprise Edition version","Version Magento Enterprise Edition"
+"Quick reference guide","Guide de référence rapide"
+"Log in to the ","Connectez-vous au "
+"Buckaroo Payment Plaza","Place de paiement Buckaroo"
+" to collect the following security settings:"," pour collecter les paramètres de sécurité suivants:"
+"Support","Soutien"
+"This extension is developed by Buckaroo.","Cette extension est développée par Buckaroo."
+"Extension basic configuration and account information","Configuration de base de l'extension et informations sur le compte"
+"Buckaroo Payment Plaza:","Place de paiement Buckaroo:"
+"Phone:","Téléphoner:"
+"E-mail:","E-mail:"
+"Extension support and advanced configuration","Prise en charge des extensions et configuration avancée"
+"Online documentation and FAQ:","Documentation en ligne et FAQ:"
+"Support question:","Questions d'assistance:"
+"Support by email:","Autres questions par e-mail:"
+"Documentation","Documentation"
+"A list of documents regarding the extension.","Une liste de documents concernant l\'extension."
```

Comment:
Backslashes are not needed here for single quote.

Reply:
@Buckaroo-Rene